app/controllers/search_controller.rb

```diff
@@ -8,7 +8,7 @@ def search
                        .paginate(page: params[:page], per_page: 25)
 
                if search_data[:search].present?
-                 posts.search(search_data[:search]).user_sort({ term: params[:search], default: :search_score },
+                 posts.search(search_data[:search]).user_sort({ term: params[:sort], default: :search_score },
                                                               relevance: :search_score, score: :score, age: :created_at)
                else
                  posts.user_sort({ term: params[:search], default: :score },
```

Comment:
This line needs to be changed to `params[:sort]` too, then this is good to go

Reply:
Done